### PR TITLE
Hugely improved Simplified Chinese translation quality. (Feature/zhimprove)

### PR DESCRIPTION
**Pull request by f0rb1d**
Hi,

I've updated Simplified Chinese language pack for Pterodactyl panel.

BTW, I couldn't find the best counterparts for "Nests" and "Eggs" in Chinese under such context. So, I didn't change the current translation for the two words. I may update in a few days or so. But it's okay if you merge it soon ASAP since I couldn't bear the current translation.

Also, I found this website named [Pterodactyl Panel Chinese Community](https://www.pterodactyl.org.cn/) and their Github account [Pterodactyl-CN](https://github.com/Pterodactyl-CN). Are them officially authorized by you guys or they are just purely made by enthusiasts?

Sorry for any inconvenience.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ This file is a running track of new features and fixes to each version of the pa
 
 This project follows [Semantic Versioning](http://semver.org) guidelines.
 
+### Updated
```
Comment (DaneEveritt) [inline CHANGELOG.md]: This would go under 0.7.12 since that version isn't released yet. :)

File: resources/lang/zh/admin/pack.php [hunk 2]
```diff
@@ -9,8 +9,8 @@
 
 return [
     'notices' => [
-        'pack_updated' => '整合包已经被更新。',
-        'pack_deleted' => '成功删除整合包： ":name" 。',
-        'pack_created' => '一个整合包已被成功创建，现在可以用它来部署服务器了。',
+        'pack_updated' => '已成功更新整合包。',
+        'pack_deleted' => '已成功从系统中删除整合包 “:name”。',
```
Comment (DaneEveritt) [inline resources/lang/zh/admin/pack.php]: Is there a reason for the special quotes here rather than standard ones?

Comment (f0rb1d) [reply resources/lang/zh/admin/pack.php]: In Chinese, we don't use standard quotation marks which are halfwidth. We often use fullwidth quotation marks like “”. And of course period (. to 。) etc.

But it's also completely fine to use standard ones or just delete all of them. So, this is more like a personal habit.

Comment (DaneEveritt) [reply resources/lang/zh/admin/pack.php]: Interesting, was curious since it wasn't consistent throughout the file, this is fine though.

Comment (f0rb1d) [reply resources/lang/zh/admin/pack.php]: Well try to fix that in next commit :D

BTW, how about that Chinese community thing?

**Comment by DaneEveritt** [comment]
As for the community, they're not an official community for Pterodactyl and have 0 affiliation with the project.

**Comment by gzzchh** [comment]
I am the principal of Pterodactyl China Community. This community have no no relationship with the official, just an community for all the fans of pterodactyl in China. You can contact me for more information.

**Comment by SaigyoujiYuyuko233** [comment]
This community created by the fans of the Pterodactyl Panel !
We don't have any affiliation this project. We just want to make it better and let this panel become more popular in China.

In this Pterodactyl Panel translation packet, the word "eggs" and "Nests", we also don't have the BEST translation of that so we are finding the BEST translation of that.We are trying to do our BEST for the Pterodactyl Panel! If you don't like that, I'm sorry about that.

**Comment by SaigyoujiYuyuko233** [comment]
This community created by the fans of the Pterodactyl Panel !
We don't have any affiliation this project. We just want to make it better and let this panel become more popular in China.

In this Pterodactyl Panel translation packet, the word "eggs" and "Nests", we also don't have the BEST translation of that so we are finding the BEST translation of that.We are trying to do our BEST for the Pterodactyl Panel! If you don't like that, I'm sorry about that.

best wishes,

**Comment by SaigyoujiYuyuko233** [comment]
This community created by the fans of the Pterodactyl Panel !
We don't have any affiliation this project. We just want to make it better and let this panel become more popular in China.

In this Pterodactyl Panel translation packet, the word "eggs" and "Nests", we also don't have the BEST translation of that so we are finding the BEST translation of that.We are trying to do our BEST for the Pterodactyl Panel! If you don't like that, I'm sorry about that.

I am the member of the Pterodactyl China Community (Maybe). If you have any question, you can contact @gzzchh or me for ask! :)

best wishes,
SaigyoujiYuyuko

**Pull request merged**